addon/components/polaris-data-table.js

```diff
@@ -262,6 +234,16 @@ export default Component.extend(ContextBoundEventListenersMixin, ContextBoundTas
    */
   totalsRowHeading: 'Totals',
 
+  /**
+   * Callback fired on click or keypress of a sortable column heading.
+   *
+   * @property onSort
+   * @type {function}
+   * @default no-op
+   * @public
+   */
+  onSort(/* headingIndex, direction */) {},
```

Comment:
This is a public property so I'd like it to stay with the other public properties